scripts/administrative_borders.py

```diff
@@ -1,11 +1,10 @@
-import click
```

Comment:
I cannot comment on the moved `__init__.py`, thus I am adding a comment here: `scripts/__init__.py` does not make sense as scripts don't require an init.py -- the init.py makes this a Python package.